config/apply/ccsm/prefetch.js

```diff
@@ -1,15 +1,12 @@
 export const prefetch = {
   'Patient': 'Patient/{{context.patientId}}',
-  'ObservationCore': 'Observation?patient={{context.patientId}}&category=core-characteristics',
-  'ObservationLab': 'Observation?patient={{context.patientId}}&category=laboratory',
-  'ObservationLabor': 'Observation?patient={{context.patientId}}&category=labor-delivery',
-  'ObservationObGyn': 'Observation?patient={{context.patientId}}&category=obstetrics-gynecology',
-  'ObservationSmart': 'Observation?patient={{context.patientId}}&category=smartdata',
-  'Condition': 'Condition?patient={{context.patientId}}',
+  'ConditionEncounterDiagnosis': 'Condition?patient={{context.patientId}}&category=encounter-diagnosis&clinical-status=active',
   'ConditionProblem': 'Condition?patient={{context.patientId}}&category=problem-list-item&clinical-status=active',
-  'MedicationRequest': 'MedicationRequest?patient={{context.patientId}}',
-  'Procedure': 'Procedure?patient={{context.patientId}}',
-  'DiagnosticReport': 'DiagnosticReport?patient={{context.patientId}}',
-  'Encounter': 'Encounter?patient={{context.patientId}}',
-  'Immunization': 'Immunization?patient={{context.patientId}}'
+  'ConditionMedicalHistory': 'Condition?patient={{context.patientId}}&category=medical-history&clinical-status=active',
+  'MedicationRequest': 'MedicationRequest?patient={{context.patientId}}&status=active&intent=order',
+  'ProcedureOrders': 'Procedure?patient={{context.patientId}}&category=103693007',
+  'ProcedureSurgerical': 'Procedure?patient={{context.patientId}}&category=387713003',
+  'DiagnosticReport': 'DiagnosticReport?patient={{context.patientId}}&category=http://terminology.hl7.org/CodeSystem/v2-0074|LAB',
+  'Encounter': 'Encounter/{{context.encounterId}}',
```

Comment:
I intended to change this to a READ instead of a SEARCH so that it just reads the current encounter (`{{context.encounterId}}`) instead of searching every encounter the patient has ever had. I'm not sure if the current encounter would necessarily have a status of finished.

Our CDS [only uses an Encounter to determine if somebody is symptomatic](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/DashboardLibrary.cql#L897) which is something that would [only be relevant if it happened within the past 24 hours due to the sympotmatic lookback](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/ScreeningSymptomaticLibrary.cql#L70C7-L70C26). Therefore, I made the assumption that only the current encounter is relevant here.

Reply:
Oh I see.